content/blog/2023-12-20-changepoint_explore.Rmd

```diff
@@ -0,0 +1,91 @@
+
+<center>
+![**Figure 1.** Total and Variant Case Counts with variant changepoints on April 24, 2021; June 19, 2021; December 18, 2021; March 5, 2022; June 18, 2022.](/blog/2023-12-20-changepoint_explore_files/image3.png)
+</center>
+
+## Methods
+
+Next, we needed to identify how these ground truth changepoints compared to changepoints in indicators (using the same three algorithms). We used all available data streams during our selected time range from Delphi’s [COVIDCast database](https://cmu-delphi.github.io/delphi-epidata/), which corresponded to 60 indicators (including the Facebook survey, Google Search, claims, hospitalization, and mortality indicators) each at the state, territory, and national level. When we aggregated the calculated changepoints per indicator per week, as Fig. 2 shows, we identified that some weeks have many more changepoints than others. To match the number of ground truth changepoints, we filter to the top 5 weeks per indicator (major indicator changepoints) to directly compare with the ground truth changepoints. 
+
+<center>
+![**Figure 2.** The aggregated count of changepoints per week in all fifty two regions by all three algorithms for estimated COVID related outpatient doctor visits](/blog/2023-12-20-changepoint_explore_files/image5.png)</center>
+
+Then, we categorized indicators as early, on-time, or late using the following definitions: 
+
+- **Early**: The five major indicator changepoints consistently precede the ground truth changepoints. 
+- **On-time**: The major indicator changepoints appear within a week of the ground truth changepoints
+- **Late**: The  major indicator changepoints lag behind the ground truth changepoints by more than a week. 
+
+As we expected each wave of the pandemic to last around four weeks, we necessitate that any major indicator changepoints found with a distance greater than 28 days to the nearest ground truth changepoint have no relationship with the ground truth changepoint.
```

Comment:
im not sure i understand this...  what do you mean by "necessitate"?  are you saying that any indicator that exhibits this behavior gets classified into "inconsistent/variable" or "undetermined" categories that you talk about later?

Reply:
I just added some clarification: 

> As we expect COVID dynamics (like waves) to last around four weeks, if any major indicator changepoints is more  than 28 days away from the nearest ground truth changepoint, it is likely a changepoint from an unrelated phenomena. Therefore, we only consider changepoints to have a leading or lagging relationship if the major indicator changepoint is within 28 days from a ground truth changepoint.


What do you think?

Reply:
much better!  id also change `changepoints is` to `changepoint is`